Blish HUD/Controls/TabbedWindow.cs

```diff
@@ -57,9 +59,14 @@ public class TabbedWindow : WindowBase {
             set => SetProperty(ref _hoveredTabIndex, value);
         }
 
-        private readonly Dictionary<WindowTab, Rectangle> _tabRegions = new Dictionary<WindowTab, Rectangle>();
-        private readonly Dictionary<WindowTab, Panel>     _panels     = new Dictionary<WindowTab, Panel>();
-        private          List<WindowTab>                  _tabs       = new List<WindowTab>();
+        private readonly LinkedList<IView> _currentNav = new LinkedList<IView>();
```

Comment:
I know this comes from `WindowBase`, but why did you choose to use `LinkedList` here instead of a `Stack`? All the methods you use below are also available (or equivalents) in `Stack<T>`